src/modules/fancyzones/lib/util.h

```diff
@@ -120,7 +120,7 @@ UINT GetDpiForMonitor(HMONITOR monitor) noexcept;
 void OrderMonitors(std::vector<std::pair<HMONITOR, RECT>>& monitorInfo);
 void SizeWindowToRect(HWND window, RECT rect) noexcept;
 
-bool IsInterestingWindow(HWND window, const std::vector<std::wstring>& excludedApps) noexcept;
+bool IsInterestingWindow(HWND window, const std::vector<std::wstring>& excludedApps, bool IsNewlyCreatedWindow = false) noexcept;
```

Comment:
I'd suggest not using the default parameter for better readability and to make it easier to see what's changed.